ls1intum/Artemis
# Bugfix/participation table spacing

### Checklist
#### General


#### Server
#### Client
#### Changes affecting Programming Exercises

### Motivation and Context


https://github.com/ls1intum/Artemis/issues/4556

### Description
Changed the div class within the table to make sure buttons are still visible when the width of the page is reduced.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student
- 1 Text Exercise

1. Log in to Artemis
2. Navigate to Participation View
3. Reduce the width of your browser and observe the button's within the table.

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage






### Screenshots


![vlcsnap-2022-01-22-18h40m35s416](https://user-images.githubusercontent.com/67344158/150645370-2c0fc43c-6e63-4998-bcdc-b2891746d3af.png)
